dynamic/clientset/dynamic_clientset.go

```diff
@@ -64,6 +64,15 @@ func (cs *Clientset) Kind(apiVersion, kind, namespace string) (*ResourceClient,
 	return cs.resource(apiResource, namespace)
 }
 
+func getResourceGVR(apiResource *dynamicdiscovery.APIResource) schema.GroupVersionResource {
```

Comment:
Let's move this to a method called `GroupVersionResource()` on `APIResource` in `dynamic/discovery`, to be consistent with `GroupVersionKind()` and make this immediately available anywhere we already have an object of this type.